pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/LiteralTransformFunction.java

```diff
@@ -269,4 +269,9 @@ public RoaringBitmap getNullBitmap(ValueBlock valueBlock) {
     bitmap.add(0L, length);
     return bitmap;
   }
+
+  // checks whether the literal is null or not.
```

Comment:
nit: The comment is redundant, it simply repeats what the function name says.

https://blog.boot.dev/clean-code/code-comments/
Section "AVOID REDUNDANT COMMENTS. STRIVE FOR A SINGLE SOURCE OF TRUTH"